app/code/community/Oyst/OneClick/controllers/PaymentController.php

```diff
@@ -29,7 +29,9 @@ public function urlAction()
         $response = $oneclickApi->authorizeOrder($rawData);
 
         $this->getResponse()->setHttpResponseCode(200);
-        $this->getResponse()->setHeader('Content-type', 'application/json');
+        if (php_sapi_name() == 'cgi-fcgi') {
```

Comment:
Could you use inverse the test and use === like:
_if ('cgi-fcgi' === php_sapi_name()) {_